docs/anchoring-assets/anchors-from-scratch/enabling-cross-border-payments/setting-up-test-server.mdx

```diff
@@ -6,11 +6,11 @@ sidebar_position: 30
 import { CodeExample } from "@site/src/components/CodeExample";
 import { Alert } from "@site/src/components/Alert";
 
-Before setting up a server that connects to live banking rails and real money, you should build out a test rig connected to the [Stellar test network](../../fundamentals-and-concepts/testnet-and-pubnet). The testnet works just like the main Stellar network, but it uses test data and allows you to fund test accounts for free using a tool called [Friendbot](../../fundamentals-and-concepts/testnet-and-pubnet#friendbot).
+Before setting up a server that connects to live banking rails and real money, you should build out a test rig connected to the [Stellar test network](https://developers.stellar.org/docs/fundamentals-and-concepts/testnet-and-pubnet#moving-your-project-from-testnet-to-production). The testnet works just like the main Stellar network, but it uses test data and allows you to fund test accounts for free using a tool called [Friendbot](https://developers.stellar.org/docs/fundamentals-and-concepts/testnet-and-pubnet#moving-your-project-from-testnet-to-production).
```

Comment:
It's better not to use final host domain. You can just replace it with `/`
(`/docs/<...>/`)